app/internal/windows/windows.go

```diff
@@ -767,8 +791,11 @@ func SetWindowPos(hwnd syscall.Handle, hwndInsertAfter uint32, x, y, dx, dy int3
 }
 
 func SetWindowText(hwnd syscall.Handle, title string) {
-	wname := syscall.StringToUTF16Ptr(title)
-	_SetWindowText.Call(uintptr(hwnd), uintptr(unsafe.Pointer(wname)))
+	wname, err := syscall.UTF16FromString(title)
+	if err != nil {
+		return
```

Comment:
I think an explicit `panic(err)` is appropriate here. Ignoring the title change is more confusing.